torch/csrc/utils/python_arg_parser.cpp

```diff
@@ -366,6 +366,22 @@ bool is_tensor_list_and_append_overloaded(PyObject* obj, std::vector<py::handle>
   return true;
 }
 
+bool is_float_list_and_append_overloaded(PyObject* obj) {
+  auto tuple = six::isTuple(obj);
+  if (!(tuple || PyList_Check(obj))) {
+    return false;
+  }
+  auto size = tuple ? PyTuple_GET_SIZE(obj) : PyList_GET_SIZE(obj);
+  for (size_t idx = 0; idx < size; idx++) {
```

Comment:
For the purpose of overload resolution, I don't think there's any reason to iterate through the entire list; you'll do that later when you actually parse the Python object.

Reply:
Done

Reply:
Also, due to this change, i had to update test_native_functions.py